code/modules/mob/mob_stat.dm

```diff
@@ -105,7 +105,9 @@
 	return tab_data
 
 /mob/proc/get_all_verbs()
-	var/list/all_verbs = deepCopyList(sorted_verbs)
+	var/list/all_verbs = new
```

Comment:
Pretty sure the `new` isn't needed.

Reply:
if you don't initialize the list it will runtime when trying to access the specific index later

Reply:
As Archanial said, Just doing `var/list/all_verbs` sets the value to `null` which'll runtime during the for-loop.

An alternative fix for this would be to call `new` on the `sorted_verbs` datum, but I don't think that follows standard conventions.